services/web/server/src/simcore_service_webserver/director_v2/_core_base.py

```diff
@@ -53,17 +53,21 @@ def _get_exception_from(
     return DirectorServiceError(status=status_code, reason=reason, url=url)
 
 
+class RetryFailedError(Exception):
+    pass
```

Comment:
Is this used somewhere?

Reply:
I see it is only defined here and not used.

Reply:
good catch thats removing